### PR TITLE
2.3

### PR DESCRIPTION
**Pull request by JasonLovesDoggo**
see release for changelog.

### CODE REVIEW

File: TTS/aws_polly.py [hunk 1]
```diff
@@ -35,11 +35,11 @@ def run(self, text, filepath, random_voice: bool = False):
         if random_voice:
             voice = self.randomvoice()
         else:
-            if not os.getenv("VOICE"):
+            if not settings.config["settings"]["tts"]["aws_polly_voice"]:
                 return ValueError(
-                    f"Please set the environment variable VOICE to a valid voice. options are: {voices}"
+                    f"Please set the TOML variable AWS_VOICE to a valid voice. options are: {voices}"
```
Comment (muhammedaksam) [inline TTS/aws_polly.py:40]: AWS_VOICE to AWS_POLLY_VOICE

File: TTS/streamlabs_polly.py [hunk 2]
```diff
@@ -35,11 +35,11 @@ def run(self, text, filepath, random_voice: bool = False):
         if random_voice:
             voice = self.randomvoice()
         else:
-            if not os.getenv("VOICE"):
+            if not settings.config["settings"]["tts"]["streamlabs_polly_voice"]:
                 return ValueError(
-                    f"Please set the environment variable VOICE to a valid voice. options are: {voices}"
+                    f"Please set the config variable STREAMLABS_VOICE to a valid voice. options are: {voices}"
```
Comment (muhammedaksam) [inline TTS/streamlabs_polly.py:40]: STREAMLABS_VOICE to STREAMLABS_POLLY_VOICE

**Review by muhammedaksam — COMMENTED** [review]
ValueErrors for aws_polly_voice & streamlabs_polly_voice

**Comment by JasonLovesDoggo** [comment]
I mean it's just a comment but good catch

**Review by CallumIO — APPROVED** [review]

**Pull request merged**